src/views/Type.vue

```diff
@@ -41,32 +40,38 @@ export default class Types extends Vue {
   public resourceType: ResourceType;
   public error: string = '';
 
-  public get resourceTypeList(): ListElement[] {
+  public get resourceTypeList(): LiOptions[] {
```

Comment:
naming - singular/plural? just a reminder to change it everywhere, if you do so.